src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerProjectSystem.cs

```diff
@@ -141,8 +156,7 @@ private async ValueTask LoadOrReloadProjectsAsync(ImmutableSegmentedList<Project
         var binaryLogPath = GetMSBuildBinaryLogPath();
 
         await using var buildHostProcessManager = new BuildHostProcessManager(binaryLogPath: binaryLogPath, loggerFactory: _loggerFactory);
-
-        var displayedToast = 0;
+        var toastErrorReporter = new ToastErrorReporter();
```

Comment:
Is it possible to have 2 (or more) calls to `LoadOrReloadProjectsAsync` for the same initial solution load?  Meaning that the toast gets displayed twice?

Reply:
_Unlikely_. We kick off the batch after a small delay, so unless the loop that adds all the projects to the batch took a long time it'd happen once.